witch hat costume for halloween party cat 🎃

leozamboni opened the pull request:
a witch hat for halloween but one day later :(

<!--
Hey there! Thanks for your contribution.

Please make sure that your commits are signed off first.
If you don't know how that works, check out our contribution guidelines: https://github.com/PrismLauncher/PrismLauncher/blob/develop/CONTRIBUTING.md#signing-your-work
If you already created your commits, you can run `git rebase --signoff develop` to retroactively sign-off all your commits and `git push --force` to override what you have pushed already.

Note that signing and signing-off are two different things!
-->

leozamboni committed:
feat: witch hat costume for halloween

leozamboni committed:
fix: formatting

leozamboni committed:
fix: formatting

leozamboni committed:
Merge branch 'develop' of github.com:leozamboni/PrismLauncher into develop

flowln commented:
You'd probably coordinate with the folks at #372 and also add a spooky Rory <:)

AshtakaOOf commented:
Should I make a Spooky Rory?

leozamboni commented:
please, themed stuff are cool

leozamboni committed:
DCO Remediation Commit for leozamboni <leonardonunes169@gmail.com>

I, leozamboni <leonardonunes169@gmail.com>, hereby add my Signed-off-by to this commit: a48c2f61ffa58601179afa44f0018d49a35d0007
I, leozamboni <leonardonunes169@gmail.com>, hereby add my Signed-off-by to this commit: d3ea43c07a2f5709f2a914e7a8687b2baf08fed4
I, leozamboni <leonardonunes169@gmail.com>, hereby add my Signed-off-by to this commit: b198053e9868983d19cc556ac09f7d730883edc8

Signed-off-by: leozamboni <leonardonunes169@gmail.com>

Scrumplex commented:
Please integrate this with the system introduced by #372

santiagocezar commented:
Basically you just need to add a `-ween` suffix (or anything else) to the cat string in the logic and save the variants with that suffix too. So most of the logic can be reused!

leozamboni committed:
fix kittehween

leozamboni committed:
feat kittehween

RaptaG commented:
Why’s this closed?

leozamboni commented:
I'm fixing

leozamboni committed:
fix cat string compare

Scrumplex commented:
In that case making this a draft is nicer I think

Scrumplex commented:
Oh. I thought I can make it a draft myself

leozamboni committed:
DCO Remediation Commit for leozamboni <leonardonunes169@gmail.com>

I, leozamboni <leonardonunes169@gmail.com>, hereby add my Signed-off-by to this commit: 7a31c56a5a21670edd517ab377eaaa143fd1f499
I, leozamboni <leonardonunes169@gmail.com>, hereby add my Signed-off-by to this commit: 00a7f81183fde74544dd28cf89af0d2b1551afdd

Signed-off-by: leozamboni <leonardonunes169@gmail.com>

leozamboni commented:
github automatically closed the pr to update my fork, I don't know why

TheKodeToad commented:
It seems the if statement's formatting is a little inconsistent.
Yes...
That's right...
Time for clang-format.

leozamboni committed:
formatting

Signed-off-by: leozamboni <leonardonunes169@gmail.com>

leozamboni commented:
done :)

leozamboni committed:
fix: block only formatting

Signed-off-by: leozamboni <leonardonunes169@gmail.com>

leozamboni commented:
- [6b856c9](https://github.com/PrismLauncher/PrismLauncher/pull/373/commits/6b856c93eabb6358bff84b6fc0c31415476d3b9b) formatted the entire file by mistake
- [498a1cd](https://github.com/PrismLauncher/PrismLauncher/pull/373/commits/498a1cdf1693710d17af8406fb95789f8be2ea89) revert and format only the block

santiagocezar commented on result:
```diff
@@ -0,0 +1 @@
+/nix/store/ps23952qj1qlvi7vm0jf2n03i9mlx8fq-prismlauncher-20221102
```
I don't think this should be here?

santiagocezar commented:
BTW you can probably just squash the changes once you're done, so don't worry about those commits

leozamboni committed:
refact: remove nix build result

Signed-off-by: leozamboni <leonardonunes169@gmail.com>

leozamboni committed:
squash: feat spooky kitteh cat

Signed-off-by: leozamboni <leonardonunes169@gmail.com>

leozamboni committed:
Merge branch 'develop' of github.com:leozamboni/PrismLauncher into develop

santiagocezar reviewed (approved):
![image](https://user-images.githubusercontent.com/48573316/199823026-d7414147-3791-4a90-9190-34e33d596f6d.png)

hat cat 👍

AshtakaOOf reviewed (approved):
Cat are cool so I approve :+1:

Scrumplex reviewed (approved):
LGTM

Scrumplex commented:
@leozamboni Can you squash all your commits into one? We could do that when merging on GitHub, but that would replace your own signature with GitHub's signature. If you need help, do not hesitate to ask.
